backend/src/packages/users/user.repository.ts

```diff
@@ -28,6 +28,14 @@ class UserRepository implements IRepository {
     return users.map((it) => UserEntity.initialize(it));
   }
 
+  public async findById(id: number): Promise<UserEntity | null> {
+    const user = await this.userModel.query().findById(id);
+    if (!user) {
```

Comment:
It is a good practise to divide logical blocks with empty spaces, which will increase readabillity.
Add empty space below and above if statement.